app/classifier.py

```diff
@@ -11,7 +11,7 @@
 import numpy as np
 import tensorflow as tf
 
-from app import embedder
+from app import embedder as e
```

Comment:
maybe better from app.embedder import Embedder? e. is used once and smells like Go ;)

Reply:
done